Login/js/form.js

```diff
@@ -0,0 +1,13 @@
+function swap() {
+  var x = document.getElementById("login-form");
+    x.style.display = "none";
+  var y = document.getElementById("forgot-pw-form");
+  	y.style.display = "block";
+}
+
+function swap_back(){
+	var x = document.getElementById("login-form");
+	  x.style.display = "block";
+	var y = document.getElementById("forgot-pw-form");
+		y.style.display = "none";
+}

```

Comment:
No other validation at this point? Not block worthy cause we are not dealing with web at this point but worth mentioning